src/styles/index.scss

```diff
@@ -1,3 +1,220 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+  --column-count: 1;
+}
+
+.imgblock {
+  height: 40px;
+  width: 40px;
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  color: inherit;
+}
+
+nav {
+  align-content: center;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  font-family: Roboto, sans-serif;
+  height: var(--header-height);
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navItems {
+  display: flex;
+  &__block {
+    font-style: normal;
+    align-content: center;
+    margin-left: 20px;
+    &:first-child {
+      margin: 0;
+    }
+    & > a {
+      font-size: 12px;
+      font-weight: 500;
+      height: var(--header-height);
+      align-content: center;
+      color: #060b35;
+      transition: 0.3s;
+      &:hover {
+        color: var(--link-color);
+      }
+    }
+  }
+}
+
+.is-active {
+  color: var(--link-color) !important;
+  position: relative;
+}
```

Comment:
The '.is-active' class is correctly styled to change the color to the link color and position it relatively, as specified in the task.